ddprof-lib/src/main/cpp/vmEntry.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2016 Andrei Pangin
+ * Copyright 2021, 2023 Datadog, Inc
```

Comment:
If we aren't adding anything to the file, should we put a copyright notice on it?

Reply:
Yeah, this is catch-up .. it should have been updated with the previous changes. Now they are rolled back so this looks like a no-op.